src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,10 @@
+db.name=disc_golf_PROD
+db.user=dg_user_PROD
+db.password=${DISC_GOLF_DB_PASS_PROD}
+
+server.port: 8443
+security.require-ssl=true
```

Comment:
We don't have SSL certificate for now for this service. Let's remove ssl cofnig (including server.port).

Reply:
Fixed